app/views/weight_estimator/index.html.erb

```diff
@@ -15,10 +17,9 @@
           </div>
           <div class="usa-width-three-fourths">
             <div class="usa-grid-full hhg-category-section-one">
-              <%- @household_goods = category.household_goods.order(:name) -%>
-              <%- @half = @household_goods.length / 2 -%>
-              <%= render partial: 'household_goods', object: @household_goods.first(category.household_goods.length - @half) %>
-              <%= render partial: 'household_goods', object: @household_goods.last(@half) %>
+              <%- category.household_goods.in_groups(2, false).each do |household_goods_group| -%>
+                <%= render partial: 'household_goods', object: household_goods_group %>
+              <% end %>
```

Comment:
So this is a fun one.

Ruby's got this [`in_groups` method](https://apidock.com/rails/Array/in_groups) that'll split up an array into any number of arrays you pass it. The second argument (set to `false` in our case) tells Ruby whether or not to pad out remaining slots.

Reply:
oh useful